packages/preview/universal-cau-thesis/0.1.0/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "universal-cau-thesis"
+version = "0.1.0"
+entrypoint = "sample.typ"

```

Comment:
The structure of your package will not work as intended. You should normally have some files that contain all layout and utility functions (here it seems to be your `ref` folder), that will be the entrypoint for the package. Then, you should have a sub-directory that contains the template itself, that is the files that will be copied to new projects based on this template. The template should depend on the package.

Reply:
Thanks, I misunderstood that. I have already changed the entrypoint, and tested my package locally on my system and it worked.